contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics.py

```diff
@@ -77,6 +77,38 @@ def get_process_private_bytes_metric():
     return gauge
 
 
+def get_process_cpu_usage():
+    try:
+        # In the case of a process running on multiple threads on different CPU
+        # cores, the returned value of cpu_percent() can be > 100.0. The actual
+        # value that is return from this function should be capped at 100.
+        return min(PROCESS.cpu_percent(), 100.0)
```

Comment:
Please check with Cijo Thomas on how we should handle this in a consistent way, probably we need to normalize against the cores.

Reply:
Right. AI-Node SDK does this too.